src/content/docs/en/reference/components-reference.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: Built-in Components Reference
+i18nReady: true
+---
+import Since from '~/components/Since.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import ReadMore from '~/components/ReadMore.astro';
+
+## Built-in Components
+
+Astro includes several built-in components for you to use in your projects. All built-in components are available in `.astro` files via `import {} from 'astro:components';`.
```

Comment:
```suggestion
Astro includes several built-in components for you to use in your projects. All built-in components are available in `.astro` files via `import {} from 'astro:components'`.
```
Not sure if the semi is necessary in an inline code example, but it's been a while!

Reply:
Agreed. Looks a little funky.